src/Compiler/CompilerOpenFPGA.cpp

```diff
@@ -492,6 +492,25 @@ bool CompilerOpenFPGA::RegisterCommands(TclInterpreter* interp,
   };
   interp->registerCmd("flat_routing", flat_routing, this, 0);
 
+  auto use_vpr_latest = [](void* clientData, Tcl_Interp* interp, int argc,
+                           const char* argv[]) -> int {
+    CompilerOpenFPGA& comp = *(CompilerOpenFPGA*)clientData;
+    std::filesystem::path vpr = comp.m_vprExecutablePath.lexically_normal();
+    constexpr bool dbg = false;
+    if (dbg) std::cerr << "\n  VPR WAS: " << vpr << '\n';
+    try {
+      vpr.replace_filename("vpr_latest");
+      comp.VprExecPath(vpr);
+      if (dbg) std::cerr << "  VPR NOW: " << vpr << "\n\n";
+    } catch (...) {
+      if (dbg) {
+        assert(0);
```

Comment:
Why do we need a assert and a dgb flag ?

Reply:
replace_filename may throw exceptions according to
https://en.cppreference.com/w/cpp/filesystem/path/replace_filename

assert(0) make it easy to debug the exception situation

'dbg' is not really a flag, it's a way to comment out code. It's constexpr false at compile time so the parts after if(dbg) are not compiled, it's equivalent to commenting out. I'll remove this stuff.

Reply:
Got it, thanks.
We don't have a way to handle debug messages yet. You can use the ErrorMessage option as requested.